### DISCUSSION TITLE
[mini.pairs] How can I make `mini.pairs` smarter?

### DISCUSSION THREAD

**Discussion by toshmukhamedov** [nvim-mini/mini.nvim#719] [Q&A] [answered]
Is it possible to configure `mini.pairs` to behave like `ZhiyuanLck/smart-pairs`?

**Comment by echasnovski** [comment, marked as answer]
Maybe, to a certain degree. Here is a [list of supported features](https://github.com/echasnovski/mini.nvim/blob/b7403ad0c2a4dab777244171ca1b7e8c89696584/doc/mini-pairs.txt#L8-L32) and, more importantly, what is not and will not be supported.
